### PR TITLE
Restart grafana during chroma-config setup

### LINKED ISSUE #1095: grafana.service should restart on `chroma-config setup`
The `grafana.service` should be restarted within `chroma-config setup` so any provisioning changes made can take effect within grafana.

### PR DESCRIPTION
**Pull request by utopiabound**
Fixes #1095

Signed-off-by: Nathaniel Clark <nclark@whamcloud.com>

### CODE REVIEW

File: chroma_core/lib/service_config.py [hunk 1]
```diff
@@ -293,7 +293,7 @@ def _setup_rabbitmq_credentials(self):
         self.try_shell(sudo + ["rabbitmqctl", "set_user_tags", settings.AMQP_BROKER_USER, "management"])
 
     def _setup_influxdb(self):
-        influx_service = ServiceControlEL7("influxdb")
+        influx_service = ServiceControl("influxdb")
```
Comment (johnsonw) [inline chroma_core/lib/service_config.py]: Should we also restart influx when restarting chroma? I was looking at postgres and I see the following:

```
 def _restart_pgsql(self):
        postgresql_service = ServiceControl.create("postgresql")
        if postgresql_service.running:
            postgresql_service.reload()
        else:
            postgresql_service.start()
        postgresql_service.enable()
```

Comment (utopiabound) [reply chroma_core/lib/service_config.py]: Influx does restart during setup

Comment (johnsonw) [reply chroma_core/lib/service_config.py]: ok guess I missed that.

**Comment by jgrund** [comment]
@utopiabound Looks like there is a different bug now.

**Pull request merged**